src/lib/config.js

```diff
@@ -44,8 +58,34 @@ function loadConfig () {
       }
     }
   }
+}
+
+function validateConfig () {
+  // Validate precision
+  const commonConfig = Config.loadConfig(envPrefix)
+  const isOracle = commonConfig.get('db.uri').match(/oracle/) !== null
```

Comment:
Might be safer to test `startsWith('oracle://')`, the server name could contain "oracle".